docs/data/base/pages.ts

```diff
@@ -25,7 +25,7 @@ const pages: readonly MuiPage[] = [
           { pathname: '/base-ui/react-button', title: 'Button' },
           { pathname: '/base-ui/react-checkbox', title: 'Checkbox', planned: true },
           { pathname: '/base-ui/react-input', title: 'Input' },
-          { pathname: '/base-ui/react-number-input', title: 'Number Input' },
+          { pathname: '/base-ui/react-number-input', title: 'Number Input', newFeature: true },
```

Comment:
Let's remove this and go with the solution from #38573 when it's merged.

Reply:
Updated!